docs/storage.md

```diff
@@ -0,0 +1,64 @@
+# Storage Device Selection
+
+This tutorial illustrates the storage device options in `Acquire`.
+
+## Description of Storage Devices
+
+To start, we'll create a `Runtime` object and print the storage device options.
+
+```python
+import acquire 
+
+# Instantiate a Runtime object
+runtime = acquire.Runtime()
+
+# Instantiate a DeviceManager object for the Runtime
+manager = runtime.device_manager()
+
+# Print devices in DeviceManager of kind Storage
+for device in manager.devices():
+    if device.kind == acquire.DeviceKind.Storage:
+        print(device)
+```
+The output of that script will be:
+
+```
+# Storage Devices printed
+
+<DeviceIdentifier Storage "raw">
+<DeviceIdentifier Storage "tiff">
+<DeviceIdentifier Storage "trash">
+<DeviceIdentifier Storage "tiff-json">
+<DeviceIdentifier Storage "Zarr">
+<DeviceIdentifier Storage "ZarrBlosc1ZstdByteShuffle">
+<DeviceIdentifier Storage "ZarrBlosc1Lz4ByteShuffle">
+```
+- **raw** - Streams to a raw binary file.
+- **tiff** - Streams to a [bigtiff](http://bigtiff.org/) file. Metadata is stored in the `ImageDescription` tag for each frame as a `JSON` string.
+- **trash** - Writes nothing. Discards incoming data. Useful for live streaming applications.
+- **tiff-json** - Stores the video stream in a [bigtiff](http://bigtiff.org/), and stores metadata in a `JSON` file. Both are located in a folder identified by the `filename` property.
+- **Zarr** - Streams data to a [Zarr V2](https://zarr.readthedocs.io/en/stable/spec/v2.html) file with associated metadata.
+- **ZarrBlosc1ZstdByteShuffle** - Streams compressed data (_zstd_ codec) to a [Zarr V2](https://zarr.readthedocs.io/en/stable/spec/v2.html) file with associated metadata. 
+- **ZarrBlosc1Lz4ByteShuffle** - Streams compressed data (_lz4_ codec) to a [Zarr V2](https://zarr.readthedocs.io/en/stable/spec/v2.html) file with associated metadata.
+
+`Acquire` supports streaming data to [bigtiff](http://bigtiff.org/) and [Zarr V2](https://zarr.readthedocs.io/en/stable/spec/v2.html). 
+
+Zarr has additional capabilities relative to the basic storage devices, namely _chunking_, _compression_, and _multiscale storage_. To compress while streaming, you can use one of the `ZarrBlosc1*` devices. Compression is done via [Blosc](https://www.blosc.org/pages/blosc-in-depth/).
+Supported codecs are _lz4_ and _zstd_, available with **ZarrBlosc1Lz4ByteShuffle** and **ZarrBlosc1ZstdByteShuffle** devices, respectively. For a comparison of these codecs, please refer to the [Blosc docs](https://www.blosc.org/). To enable _chunking_ and/or _multiscale storage_, set those attributes in an instance of the `ChunkingProperties` and `StorageProperties` classes, respectively. You can learn more about the Zarr capabilities in `Acquire` [here](https://github.com/acquire-project/acquire-driver-zarr).

```

Comment:
```suggestion
Zarr has additional capabilities relative to the basic storage devices, namely _chunking_, _compression_, and _multiscale storage_. You can learn more about the Zarr capabilities in `Acquire` [here](https://github.com/acquire-project/acquire-driver-zarr).
```

Since we don't actually use these capabilities, instead of describing them in detail like this it's fine to just point to the Zarr driver if they want to learn more.